src/Mod/PartDesign/Gui/TaskLinearPatternParameters.cpp

```diff
@@ -210,15 +222,28 @@ void TaskLinearPatternParameters::updateUI()
         dirLinks.setCurrentLink(pcLinearPattern->Direction);
     }
 
-    // Note: These three lines would trigger onLength(), on Occurrences() and another updateUI() if we
+    // Note: These three lines would trigger related on*() and another updateUI() if we

```

Comment:
Is this sentence correct ?

Reply:
I'm not sure anymore ;) I've changed this comment to:
```
    // Note: This block of code would trigger change signal handlers (e.g. onOccurrences())
    // and another updateUI() if we didn't check for blockUpdate
```

Now it should be clear and easy to extend as it does not explicitly mention triggered methods.